components/navigable-container/index.js

```diff
@@ -65,10 +65,14 @@ class NavigableContainer extends Component {
 			this.props.onKeyDown( event );
 		}
 
+		if ( this.props.disabled ) {
+			return;
+		}
```

Comment:
Added a way to disable the tabbableContainer behavior without replacing the component (to avoid rerenders)